packages/api/src/APIProvider.tsx

```diff
@@ -37,7 +37,10 @@ const getDerivAPIInstance = (): DerivAPIBasic => {
     }
 
     if (!window.DerivAPI?.[wss]) {
-        window.DerivAPI[wss] = new DerivAPIBasic({ connection: new WebSocket(wss) });
+        const DerivAPIBasicInstance = new DerivAPIBasic({ connection: new WebSocket(wss) });
+        window.DerivAPI[wss] = DerivAPIBasicInstance;
+        // add this to be consistent with socket_base
+        window.DerivAPI[wss].get = () => DerivAPIBasicInstance;
```

Comment:
What is this? Why do we need to wrap this in a function. You can surely just get the instance by `window.DerivAPI[wss]`

I don't think this file should be changing.

Reply:
@markwylde-deriv As I mentioned in comment it's to be consistent with socket_base. Check the images attached

<img width="919" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/3049b133-9aca-4e29-af6f-7e9dd5ee166b">
<img width="648" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/63b9e22b-16ac-459e-b33a-86dea4d6ee0f">

Reply:
Would it be a lot of work to fix this in the socket_base, so it's consistent in both?

Reply:
@markwylde-deriv I'm pretty sure we don't need to modify `socket_base`. It's why I've added `get` method for `getDerivAPIInstance` like `get` in `socket_base`, because I don't know what `WS` connection it will be, legacy `WS` or `WS` from `APIProvider`